test/e2e/serial/tests/configuration.go

```diff
@@ -118,11 +118,12 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 			Expect(err).ToNot(HaveOccurred())
 
 			defer func() {
-				By(fmt.Sprintf("unlabeling node: %q", targetedNode.Name))
+				By(fmt.Sprintf("Restore initial labels of node %q", targetedNode.Name))
+				klog.Infof("unlabeling node: %q with %q", targetedNode.Name, nodes.GetLabelRoleMCPTest())
 				err = unlabelFunc()
 				Expect(err).ToNot(HaveOccurred())
 
-				By(fmt.Sprintf("labeling node: %q", targetedNode.Name))
+				klog.Infof("labeling node: %q with %q", targetedNode.Name, nodes.GetLabelRoleWorker())
```

Comment:
let's keep the `By` which is actually more visible than klog. The enhanced log is fine.